src/app/components/channel-actions/channel-actions.component.html

```diff
@@ -14,10 +14,20 @@
     mat-button
     color="accent"
     [disabled]="channel.state !== 'opened'"
-    (click)="openDepositDialog.emit(true)"
+    (click)="deposit()"
 >
     {{ 'Deposit' | uppercase }}
 </button>
+<button
+    class="match-parent-width"
+    id="withdraw-button"
+    mat-button
+    color="accent"
+    [disabled]="channel.state !== 'opened'"
+    (click)="withdraw()"
+>
+    {{ 'Withdraw' | uppercase }}
```

Comment:
It's ok for now, but this kind of thing is easier to be done via CSS, so it doesn't incurr template overhead. I think there's even some css helper classes to ease it.

Reply:
Yes, I think there should be some classes already there. I will fix it within the day. Thanks.